packages/authentication/src/authenticate.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
```

Comment:
I think we should change it for all files in `loopback-next` as you suggested, to IBM Corp. 2017 as you suggested.